app/views/components/upload/YouHaveUploadedFilesPage.scala.html

```diff
@@ -18,8 +18,8 @@
     @formHelper(action = postAction) {
         <h1 class="govuk-heading-l"> @messages(pageHeadingAndTitle) </h1>
         @radios(form = form,
-            legend = messages("uploadAnotherFile"),
-            items = radioOptions,
+            legend = if(filesUploaded < 5) messages("otherReason.uploadList.uploadAnotherFile") else "",
+            items = radioOptions ,
             hint = None,
             legendAsHeading = false,
             legendClasses = "govuk-fieldset__legend--m"
```

Comment:
I would wrap the radio field in an if statement.
```suggestion
 @if(filesUploaded < 5) {
     @radios(form = form,
            legend = messages("otherReason.uploadList.uploadAnotherFile"),
            items = radioOptions ,
            hint = None,
            legendAsHeading = false,
            legendClasses = "govuk-fieldset__legend--m"
     )
 }
```